### DISCUSSION TITLE
Corrupt Backups

### DISCUSSION THREAD

**Discussion by smccloud** [JonathanTreffler/backblaze-personal-wine-container#82] [General]
While browsing Reddit I came across this.  Has anyone else noticed it?  I have experienced the issue and I haven't even finished backing up all my files for the first time.
https://www.reddit.com/r/backblaze/comments/18e1cgm/is_your_backup_corrupt_please_check/

**Comment by traktuner** [comment]
I have not experienced this issue personally (using backblaze for 3 years+ now).
Every few months I do a test-restore with some old and new files from the backblaze site, every time successfully.
